package.json

```diff
@@ -13,6 +13,9 @@
     {
       "name": "Ricardo Bin",
       "email": "ricardohbin@gmail.com"
+    },
+    {
+      "name": "mbieser"
```

Comment:
this is a core commiter list, if everyone who contributed once the list would be too big :/

Reply:
Removed.